core/src/main/java/com/radixdlt/consensus/bft/processor/BFTEventStatelessVerifier.java

```diff
@@ -182,15 +186,20 @@ public void processVote(Vote vote) {
 
   @Override
   public void processProposal(Proposal proposal) {
-    if (!isAuthorInValidatorSet(proposal)) {
+    final var expectedAuthor = proposerElection.getProposer(proposal.getRound());
+    if (!proposal.getAuthor().equals(expectedAuthor)) {
```

Comment:
Looks like `isAuthorInValidatorSet()` can be removed?

Reply:
we still use that for verifying votes